tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "baseUrl": "app/javascript",
+    "allowSyntheticDefaultImports": true,
+    "module": "es6",
+    "lib": ["es6", "dom"],
+    "moduleResolution": "node",
+    "noImplicitAny": false,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "outDir": "dist",
+    "pretty": true,
+    "skipLibCheck": true,
+    "sourceMap": true,
```

Comment:
i think `noEmit` can be set to `true` and then `sourceMap: false` and `outDir` ommitted. That should increase perf since everything is being handled through babel.